qa-tests-backend/src/main/groovy/objects/Rsyslog.groovy

```diff
@@ -0,0 +1,56 @@
+package util
+
+import groovy.util.logging.Slf4j
+import objects.Deployment
+import objects.Service
+import orchestratormanager.OrchestratorMain
+
+@Slf4j
+class Rsyslog {
+    public static final Integer SYSLOG_PORT = 514
+    private Rsyslog() { }
+    private Service syslogSvc
+    private Deployment  deployment
+
+    static Rsyslog createRsyslog(OrchestratorMain orchestrator, String namespace) {
+        def deploymentName = "rsyslog"
```

Comment:
I always try to make the deployment name unique to be resilient to cleanup issues - though it will not matter in this particular case unless this test is being debugged against a GKE cluster for crashes and such.